capra_audio/package.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>capra_audio</name>
+  <version>0.0.0</version>
+  <description>The capra_audio package</description>
+  <maintainer email="info@clubcapra.com">Club Capra</maintainer>
```

Comment:
I don't think this email address actually exists. You might want to use `capra@clubcapra.com`